django_pydantic_field/compat/pydantic.py

```diff
@@ -1,6 +1,5 @@
-from pydantic.version import VERSION as PYDANTIC_VERSION
-
 __all__ = ("PYDANTIC_V2", "PYDANTIC_V1", "PYDANTIC_VERSION")
 
+PYDANTIC_VERSION = "1.10"
 PYDANTIC_V2 = PYDANTIC_VERSION.startswith("2.")
 PYDANTIC_V1 = PYDANTIC_VERSION.startswith("1.")
```

Comment:
I would do
```python
PYDANTIC_V2 = False
PYDANTIC_V1 = True
```
It's simpler and there is no need to specify a version that we're not using

Reply:
Sure. My original rationale is that it was make the PR less destructive/easier to review. But I agree, it's a bit hacky. Updated.